relay-event-derive/src/lib.rs

```diff
@@ -440,6 +440,7 @@ impl FieldAttrs {
 }
 
 #[derive(Copy, Clone, Default)]
+#[allow(dead_code)]
```

Comment:
Is this actually still used, Rust Analyzer tells me it's unused

Reply:
That's a question, do we want to keep this around or just delete?

Reply:
If it is actually unused, I'd delete it.

Reply:
Makes sense. 
Done in [`57550e6`](https://github.com/getsentry/relay/pull/3092/commits/57550e6fad22a52509c83d2489d00501391acccd)